### ISSUE TITLE
Cannot establish connection to Homebridge Config UI

### ISSUE THREAD

**Issue by robungar** [oznu/docker-homebridge#129]
Hi, apologies in advance, but pretty new to using Docker. I installed the spk and it looks to have set up successfully - Docker says it's running and I have the icon to launch the Homebridge Config UI in my packages menu. When I click to launch it, though, it opens a new browser tab and says it cannot connect because it can't establish a secure connection. I have opened the port and confirm that it's open from my router, from my DiskStation, and from an open port checking tool:

<img width="724" alt="screen shot 2018-09-07 at 9 20 09 am" src="https://user-images.githubusercontent.com/3894093/45221916-f9204600-b280-11e8-914d-7f14001e8350.png">

<img width="1160" alt="screen shot 2018-09-07 at 9 18 52 am" src="https://user-images.githubusercontent.com/3894093/45221928-ffaebd80-b280-11e8-8f98-724f36bc9212.png">

<img width="175" alt="screen shot 2018-09-07 at 9 18 54 am" src="https://user-images.githubusercontent.com/3894093/45221946-0a695280-b281-11e8-840b-7e1645f9bf78.png">

Can anyone provide some insight? I'm sure I'm doing something wrong but the ports seem to be open. I am using my custom domain and not on the local network right this moment. But I have many applications running and have no problem accessing them from outside my network.

**Comment by oznu** [comment]
Can you post the full error in the browser?

Also try going to the site using `http://<NAS IP>:8581`.

**Comment by robungar** [comment]
Hi, I was able to get it started on my home network. I installed a plugin, restarted homebridge, and it has continued to show as running - however, I cannot access the interface. I am clicking the homebridge icon in the packages menu on DSM and have also tried manually going to my ip:8581 and ip:51826 with no success:

![screen shot 2018-09-08 at 11 46 03 am](https://user-images.githubusercontent.com/3894093/45255932-e54a1200-b35c-11e8-8635-45590a19982d.png)

![screen shot 2018-09-08 at 11 45 56 am](https://user-images.githubusercontent.com/3894093/45255934-e9762f80-b35c-11e8-8418-5e45455db8f4.png)

**Comment by robungar** [comment]
Now it keeps showing this:

![screen shot 2018-09-08 at 5 46 59 pm](https://user-images.githubusercontent.com/3894093/45258916-45f24280-b38f-11e8-8ba7-9439b3e5484f.png)

**Comment by robungar** [comment]
I seem to have it temporarily working. I reinstalled it and manually went in to change the port in the config file to 8581. Before that it was set as 8080. Why is it set as that by default if it's supposed to be 8581?

**Comment by robungar** [comment]
OK here's the problem now - I installed a plugin and went to restart Homebridge. But it just will not restart - it keeps failing:

![screen shot 2018-09-10 at 7 34 32 pm](https://user-images.githubusercontent.com/3894093/45329945-ec297e00-b530-11e8-801f-1ce586b6b5cc.png)

It just fails and clicking on the Homebridge icon in my packages brings me back to the problem before. I solved it the first time by completely uninstalling and reinstalling the package. But surely that can't be the solution every time I need to update it, right?
